play/src/main/scala/com/managementapp/authentication/models/UserTable.scala

```diff
@@ -0,0 +1,15 @@
+package com.managementapp.authentication.models
+
+import slick.lifted.Tag
+import slick.driver.MySQLDriver.api._
+
+
+case class User(email: String, password: String)
```

Comment:
Oh and DTO and domain objects and/or DAO objects should always always always be separated.
DTO should just be a thin layer between http request layer and domain layer. Because DTO can not have custom methods defined for them(They are generated code).